game/shared/ehandle.h

```diff
@@ -11,31 +11,19 @@
 #pragma once
 #endif
 
-#if defined( _DEBUG ) && defined( GAME_DLL )
-#include "tier0/dbg.h"
-#include "cbase.h"
-#endif
-
-
-#include "const.h"
-#include "basehandle.h"
-#include "entitylist_base.h"
-
-
-class IHandleEntity;
-
+#include "entityhandle.h"
+#include "entitysystem.h"
 
 // -------------------------------------------------------------------------------------------------- //
 // Game-code CBaseHandle implementation.
 // -------------------------------------------------------------------------------------------------- //
 
-inline IHandleEntity* CBaseHandle::Get() const
+inline IHandleEntity* CEntityHandle::GetHandleEntity() const

```

Comment:
Don't think renaming it is a good idea, as CHandle relies on the naming scheme that was here before, so either correct these in the CHandle, or change this one back

Reply:
Changed back to `Get()` now